backend/Tournament/views.py

```diff
@@ -51,34 +51,32 @@ def get(self, request, tournament_id):
 		return Response(serializer.data, status=status.HTTP_200_OK)
 
 	@method_decorator(must_be_authenticated)
-	@method_decorator(validate_tournament_request([TournamentState.LOBBY, TournamentState.IN_PROGRESS]))
+	@method_decorator(validate_tournament_request([TournamentState.LOBBY]))
 	def post(self, request, tournament_id):
```

Comment:
am working with the API now and understand your thought process about the api and internal tournament management now. Instead of a post here,  I would suggest using a PATCH or PUT request to start the match. POST is for creating a new resource of the kind as given by the last url parameter (like POST tournaments/ -> new tournament, POST user/ -> new user etc). I would prefer PATCH, since PUT would require me to send all information of the tournament again.  With PATCH  one would typically send a body with the fields to update in the requests, but since we don't want the user to be fully in control (although, it could be nice to have the client send a request stating that the match has been aborted or so) of the tournament state, I suggest the client sends a PATCH /tournaments/<tournament_id> and the state is changed from LOBBY -> IN_PROGRESS and is rejected for when the tournament state is in any other and transitions to them as the games unfold. If we feel like it a PATCH in an IN_PROGRESS state could go to ABORTED.
 
 Not for this PR, but the POST doesn't seem right. A POST to tournaments/<id>/matches/ on the other hand, I could live with. The response could then contain the match id, on which the client could do a get or something. I haven't fully thought this through yet. Just thoughts

Reply:
I've updated it now so that when the tournament.state == LOBBY, the tournament starts, and if the tournament.state == IN_PROGRESS, the tournament is aborted

Reply:
also changed it to PATCH of course

Reply:
So do we want to send from the frontend that a tournament is aborted? Like in between games  when no websocket is open and the client navigates away?